app/src/http/AuthHttpClient.js

```diff
@@ -3,6 +3,7 @@ import Util from '../Util';
 import config from '../config.json';
 import BlockingAdapter from './BlockingAdapter';
 import base64js from 'base64-js';
+import { deprecate } from 'util';
```

Comment:
what is this?

Reply:
rm, auto import when I used deprecate in comments